native/internal/math32/sqrt.go

```diff
@@ -0,0 +1,24 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//+build !amd64 noasm
+
+package math32
+
+import (
+	"math"
+)
+
+// Sqrt returns the square root of x.
+//
+// Special cases are:
+//	Sqrt(+Inf) = +Inf
+//	Sqrt(±0) = ±0
+//	Sqrt(x < 0) = NaN
+//	Sqrt(NaN) = NaN
+func Sqrt(x float32) float32 {
+	// FIXME(kortschak): Direct translation of the math
```

Comment:
I no longer understand this comment. You have the directly translated version in math32_test.go which you're testing against the exported Sqrt. This at least deserves a comment on why sqrt in math32_test.go is not used here.